ui/src/common/layouts/Header/InterventionBanner.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { Alert } from 'antd';
+
+import ExternalLink from '../../components/ExternalLink';
+import { getConfigFor } from '../../config';
+
+class InterventionBanner extends Component {
+  render() {
+    const interventionConfig =
+      getConfigFor('REACT_APP_INTERVENTION_BANNER') || null;
```

Comment:
maybe it would be nice to add `notSetValue ` param for `getConfigFor`. I think it will be needed later on.

```js
export function getConfigFor(configKey, notSetValue = null) {
  try {
    return window.CONFIG[configKey] || notSetValue;
  } catch (error) {
    Sentry.captureException(error);
    console.error(error); // eslint-disable-line no-console
    return notSetValue;
  }
}
```